### DISCUSSION TITLE
皮肤制作

### DISCUSSION THREAD

**Discussion by YunzeWu** [Meloong-Git/PCL#2582] [讨论]
请问龙猫可不可以在PCL2中加入皮肤自助制作功能

**Comment by iPodtouch6557** [comment]
这超出了启动器的服务范畴，并且会拖慢更新周期，大概率不会做
不知道龙猫怎么想

**Comment by WForst-Breeze** [comment]
PCL 是启动器，不是工具箱啊（笑哭）想制作皮肤右转可以去隔壁 MCSKIN3D 或者 Skinseed

**Comment by SALTWOOD** [comment]
PCL2是Minecraft启动器，不是PCL2社区/PCL2工具箱
龙猫百分百不会做，但是像你这样先在discussion问了再决定要不要投issue挺好的
做了也耗时费力，而且不是有了这个功能所有人就都会画皮肤的，面向群体窄，投入高回报低，绝对不会做

**Comment by so1ve** [comment]
照这么发展下去之后是不是会冒出来Plain Craft Operation System

**Comment by SuoHouDaiShi** [comment]
PCL事一个一个启动器，而不事一个一个工具箱
要是想做皮肤请自行搜索MCSkin3D

**Comment by iPodtouch6557** [comment]
以后再有皮肤制作一类的还是直接lock吧……真的

**Comment by Chiloven945** [comment]
真的要做皮肤出门右转一条街全是做皮肤的，你拿blockbench做皮肤我都不说你，但是你让pcl来做皮肤未免真的有点强人所难了……
